src/app/components/overview-history-summary-header/overview-history-summary-header.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, Input, OnInit } from "@angular/core";
+import {
+    Account,
+    MetadataBlockFragment,
+    Scalars,
+} from "src/app/api/kamu.graphql.interface";
+import AppValues from "src/app/common/app.values";
+import { DataHelpers } from "src/app/common/data.helpers";
+import { TableSourceInterface } from "../dynamic-table/dynamic-table.interface";
+
+@Component({
+    selector: "app-overview-history-summary-header",
+    templateUrl: "./overview-history-summary-header.component.html",
+    styleUrls: ["./overview-history-summary-header.component.sass"],
+})
+export class OverviewHistorySummaryHeaderComponent implements OnInit {
+    @Input() public tableSource?: TableSourceInterface;
+    @Input() public resultUnitText: string;
+    @Input() public metadataBlockFragment?: MetadataBlockFragment;
+    @Input() public hasResultQuantity?: boolean = false;
+    @Input() public numBlocksTotal?: number;
```

Comment:
Number does not need to be optional, the overview component ensured 0 by default.